### PR TITLE
Fix TLS listen for checking peer certificates

### PR DESCRIPTION
**Pull request by otaviolcs3**

### CODE REVIEW

File: spiffetls/tests/check_dial.c [hunk 1]
```diff
@@ -23,16 +23,16 @@ void test_TLSClientWithRawConfig(void)
...
+    ck_assert_uint_ne(err, NO_ERROR);
+    ck_assert_ptr_eq(conn, NULL);
 
     spiffeid_TrustDomain_Free(&td);
     spiffetls_DialMode_Free(mode);
 
-    const int fd = SSL_get_fd(conn);
+    /* const int fd = SSL_get_fd(conn);
     SSL_shutdown(conn);
     SSL_free(conn);
```
Comment (arianalima) [inline spiffetls/tests/check_dial.c:34]:
```suggestion
```

File: spiffetls/tests/check_dial.c [hunk 2]
```diff
@@ -56,10 +56,10 @@ void test_MTLSClient(void)
     spiffeid_TrustDomain_Free(&td);
     spiffetls_DialMode_Free(mode);
 
-    const int fd = SSL_get_fd(conn);
+    /*const int fd = SSL_get_fd(conn);
     SSL_shutdown(conn);
     SSL_free(conn);
-    close(fd);
+    close(fd);*/
```
Comment (arianalima) [inline spiffetls/tests/check_dial.c:62]: block of comment could be removed

File: spiffetls/tests/check_peerid.c [hunk 3]
```diff
@@ -36,7 +36,7 @@ START_TEST(test_spiffetls_PeerIDFromConn)
     SSL_shutdown(conn);
     SSL_free(conn);
     close(sock_fd);
-    spiffeid_ID_Free(&id);
+    spiffeid_ID_Free(&id);*/
```
Comment (arianalima) [inline spiffetls/tests/check_peerid.c:39]: block of comment could be removed

File: spiffetls/tests/check_peerid.c [hunk 4]
```diff
@@ -54,10 +54,10 @@ START_TEST(test_spiffetls_PeerIDFromConn)
     spiffeid_TrustDomain_Free(&td);
     spiffetls_DialMode_Free(mode);
 
-    sock_fd = SSL_get_fd(conn);
+    /*sock_fd = SSL_get_fd(conn);
     SSL_shutdown(conn);
     SSL_free(conn);
-    close(sock_fd);
+    close(sock_fd);*/
```
Comment (arianalima) [inline spiffetls/tests/check_peerid.c:60]: block of comment could be removed

**Review by arianalima — CHANGES_REQUESTED** [review]
just some comments to remove

**Review by rodrigolc — APPROVED** [review]

**Pull request merged**